webcompat/helpers.py

```diff
@@ -507,7 +507,7 @@ def add_csp(response):
         "font-src 'self'; " +
         "img-src 'self' https://www.google-analytics.com https://*.githubusercontent.com data:; " +  # nopep8
         "manifest-src 'self'; " +
-        "script-src 'self' https://www.google-analytics.com https://api.github.com; " +  # nopep8
+        "script-src 'self' 'unsafe-inline' https://www.google-analytics.com https://api.github.com; " +  # nopep8 @todo removed unsafe-inline
```

Comment:
I wonder if we should have a different policy just for this route.

Reply:
Yes, maybe, I guess, I don't know how I can do this .. do you could ?

Reply:
> I wonder if we should have a different policy just for this route.

I would prefer this, and it would be simpler than implementing nonce support.

Reply:
Looking at the JS again -- why can't we just stick it in a file in `webcompat/static/js/lib` and link to it in the template? That would mean we don't need to special case it.

@magsout is there some non-obvious reason?
(Also, it will mean you'll need to run prettier on it...).

Reply:
no reason, and yes we can. We can create dashboard.js and put it there.
I guess one of the reasons was to try to minimize the interactions with refactor branch. But I would be in favor of deploying the dashboard (once we merged everything). It seems refactor will take longer than expected.